.github/workflows/deploy.yml

```
@@ -0,0 +1,100 @@
+
+            # 기존 컨테이너 중지 후 재빌드/재기동
+            sudo docker compose -f docker-compose-prod.yml down || true
+            sudo docker compose -f docker-compose-prod.yml build --no-cache
+            sudo docker compose -f docker-compose-prod.yml up -d
+
+            # (옵션) 안 쓰는 이미지 정리
+            sudo docker image prune -f
+
+      # 6) (선택) CloudFront 캐시 무효화
+      - name: Configure AWS credentials
+        if: ${{ secrets.CF_DISTRIBUTION_ID != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)